src/hearings/containers/request-hearing/hearing-final-confirmation/hearing-final-confirmation.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import * as fromHearingStore from '../../../store';
+
+@Component({
+  selector: 'exui-hearing-final-confirmation',
+  templateUrl: './hearing-final-confirmation.component.html'
+})
+export class HearingFinalConfirmationComponent implements OnInit {
+
+  public heading: string;
+  public headingDescription: string;
+  public subheading: string;
+  public subheadingDescription: string;
+  public additionalDescription: string;
+  public caseId: string;
+
+  constructor(protected readonly hearingStore: Store<fromHearingStore.State>) {
+  }
+
+  public ngOnInit(): void {
+    this.hearingStore.pipe(select(fromHearingStore.getHearingList)).subscribe(
+      hearingList => {
+        this.caseId = hearingList.hearingListMainModel ? hearingList.hearingListMainModel.caseRef : '';
+        this.heading = 'Hearing request submitted';
+        this.headingDescription = 'Your hearing request will now be processed';
+        this.subheading = 'What happens next';
+        this.subheadingDescription = `You can <a href="/cases/case-details/${this.caseId}/hearings" class="govuk-link">view the status of this request in the hearings tab</a>.`;
```

Comment:
We need to use the ng router.navigate, this html href will cause whole browser refresh.

Reply:
I have made the changes in a different way. Please let me know what you think.
Or in other way, we can do something like I did for Global Search (https://github.com/hmcts/rpx-xui-webapp/blob/feature/global-search/src/search/containers/no-results/no-results.component.html). Just hard-code the html in the generic component and show hide based on hearing confirmation source enum. The only input in that case will be the case Id for the URL generation.